core/site_api.go

```diff
@@ -365,34 +365,46 @@ func (site *Site) SetBatteryModeExternal(mode api.BatteryMode) {
 		site.publish(keys.BatteryModeExternal, mode)
 
 		if !disable {
```

Comment:
Why/what for? For time being, external mode is still active. So no point doing this here and later in site.go